src/plugins/crypto/gcrypt_operations.h

```diff
@@ -0,0 +1,37 @@
+/***************************************************************************
+ gcrypt_operations.h  -  gcrypt cryptography interface
+ -------------------
+ begin                : Mon Aug 31 18:04:14 CEST 2015
+ copyright            : (C) 2015 by Peter Nirschl
+ email                : peter.nirschl@gmail.com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the BSD License (revised).                      *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef ELEKTRA_PLUGIN_GCRYPT_OPERATIONS_H
+#define ELEKTRA_PLUGIN_GCRYPT_OPERATIONS_H
+
+#include <kdb.h>
+
+// TODO adapt to Elektra's error handling facility
```

Comment:
Yes, just add warning/errors to src/liberror/specification and use them accordingly.